src/schema/portal/jgi_metagenomics.yaml

```diff
@@ -129,7 +129,7 @@ slots:
     name: dna_cont_well
     title: DNA well number
     comments:
-    - Required when 'plate' is selected for container type. Corner wells must be blank.
+    - Required when 'plate' is selected for container type. Corner wells must be blank/empty (A1, A12, H1, H12).
```

Comment:
I don't think we should say that the corner wells should be blank, but rather that JGI won't assay sample in corner wells. Therefore, samples with a `DNA well number` of A1, A12, H1, or H12 won't pass validation.

Also, I don't think 'DNA well number' is a good `title`, since the allowed values aren't numbers, but rather well identifiers or positions.